types/braintree/index.d.ts

```diff
@@ -135,7 +128,7 @@ declare namespace braintree {
         voidedAt: RangeFieldSearchFn<Date>;
     }) => void;
 
-    export type GatewayConfig = KeyGatewayConfig | ClientGatewayConfig | AccessTokenGatewayConfig | CustomConfig;
+    export type GatewayConfig = KeyGatewayConfig | ClientGatewayConfig | AccessTokenGatewayConfig;
```

Comment:
Disclosure that I'm not fully up to date on the braintree lib, but know the CustomConfig was added in recently here. What's the rationale for its removal?

Reply:
Ah, nevermind, I see the comment you left on the most recent PR that was merged in. Thanks for adding that.

Reply:
:) ah perfect,thanks for the fast approval!